CMakeLists.txt

```diff
@@ -918,6 +918,7 @@ else()
 	set(DOLPHIN_WC_IS_STABLE "0")
 endif()
 
+set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS "${CMAKE_SOURCE_DIR}/.git/HEAD")
```

Comment:
This isn't correct. You need to call git to find where the actual .git/HEAD file is, use `git rev-parse --git-path HEAD`